src/web-mercator-utils.js

```diff
@@ -39,6 +39,9 @@ export function scaleToZoom(scale) {
  * @return {Array} [x,y] coordinates.
  */
 export function lngLatToWorld([lng, lat], scale) {
+  assert(Number.isFinite(lng) && Number.isFinite(scale));
```

Comment:
FWIW, this does slightly change functionality as this is now guaranteed to generate exceptions, whereas previous I assume this would return `NaN`s...

Reply:
Correct, though `lngLatToWorld` is most commonly used followed by `worldTopixels` which will throw on `NaN`s